ReactApp/src/components/Editor.jsx

```diff
@@ -35,10 +37,21 @@ import { postMessage } from '../misc/Helpers';
 // Current editor (assumes can be only one instance).
 let editor = {};
 
-function Editor() {
+const POST_MOCK = {
+	id: 1,
+	type: 'post',
+};
```

Comment:
Do you consider this a temporary measure until we pass this information from the host app?

Reply:
Exactly, we should expect a `post` object from the host app, for now, I added the mock data.